src/components/LoanCollections/KivaClassicLoanCarousel.vue

```diff
@@ -22,34 +22,32 @@
 			@interact-carousel="onInteractCarousel"
 		>
 			<template v-for="(loanId, index) in augmentedLoanIds" #[`slide${index}`]>
-				<!-- Show View more Card -->
-				<router-link
-					v-if="loanId === 3"
-					:key="`view-more-card-${loanId}`"
-					class="tw-flex tw-items-center tw-h-full tw-w-full
-						hover:tw-bg-action-highlight hover:tw-text-primary-inverse tw-rounded"
-					:to="cleanUrl"
-					v-kv-track-event="[
-						'Homepage',
-						'click-carousel-view-all-category-loans',
-						`${viewAllLoansCategoryTitle}`]"
-				>
-					<div class="tw-w-full tw-text-center">
-						<h3>{{ viewAllLoansCategoryTitle }}</h3>
-					</div>
-				</router-link>
-
 				<!-- show loan card -->
 				<!-- TODO Re-implement card position analytics -->
 				<kiva-classic-basic-loan-card
-					v-else
 					:item-index="index"
 					:key="`loan-${loanId}`"
 					:loan-id="loanId"
 					:exp-label="expLabel"
 					:lend-now-button="lendNowButton"
 				/>
 			</template>
+			<!-- Show View more Card -->
+			<router-link
```

Comment:
we can just move this view more card outside of the loop of loans, and clean up the code that assigns it an arbitrary loan id of 3.

Reply:
I don't remember why were were even doing this...